src/overlays/actors/ovl_En_Hs/z_en_hs.h

```diff
@@ -8,21 +8,29 @@ struct EnHs;
 
 typedef void (*EnHsActionFunc)(struct EnHs*, GlobalContext*);
 
-#define ENHS_GET_F(thisx) ((thisx)->params & 0xF)
+#define HS_GET_EXIT_INDEX(thisx) ((thisx)->params & 0xF)
 
-#define ENHS_1 1
+#define HS_TYPE_UNK1 1
+
+// params mystery: Vanilla grog is 0xFE01
+//   0xFE00 space is never checked in Grog code
+//   at the same time, type UNK1 is only checked directly with params == 1, no &F
+//   so HS_TYPE_UNK1 is never valid and is unused, as 0xFE00 is still present even if its not doing anything else
+
+// The count of [chicks following the player]   is stored in [this->actor.home.rot.z] (incremented by 2)
+// The count of [chicks transformed into adult] is stored in [this->actor.home.rot.x] (incremented by 2)
 
 typedef struct EnHs {
     /* 0x0000 */ Actor actor;
     /* 0x0144 */ ColliderCylinder collider;
     /* 0x0190 */ SkelAnime skelAnime;
     /* 0x01D4 */ Vec3s jointTable[OBJECT_HS_LIMB_MAX];
     /* 0x0234 */ Vec3s morphTable[OBJECT_HS_LIMB_MAX];
-    /* 0x0294 */ Vec3s unk_294;
-    /* 0x029A */ Vec3s unk_29A;
-    /* 0x02A0 */ u16 unk_2A0;
-    /* 0x02A2 */ s16 unk_2A2;
-    /* 0x02A4 */ Vec3f unk_2A4[20];
+    /* 0x0294 */ Vec3s headRot;
+    /* 0x029A */ Vec3s unkRot29A; // probably not chest, as chest is same limb as lower body
```

Comment:
```suggestion
    /* 0x029A */ Vec3s unusedRot; // probably not chest, as chest is same limb as lower body
```
Probably just some unused rotation value since the function requires two (usually head and torso)